test/data/import/new/route-export.json

```diff
@@ -9,6 +9,7 @@
               "endpoint": "answer",
               "documentation": "",
               "enabled": true,
+              "randomResponse": false,
```

Comment:
Please do not modify files in `test/data/migrations`, `test/data/import/new` and `test/data/import/old`